src/extensions/default/Phoenix-markdownPreview/main.js

```diff
@@ -252,17 +271,26 @@ define(function (require, exports, module) {
         }
 
         realVisibility = isVisible;
+        var editor = $(" #editor-holder");
+
         if (isVisible) {
             if (!panel) {
                 $panel = $(panelHTML);
                 $iframe = $panel.find("#panel-markdown-preview-frame");
 
                 panel = WorkspaceManager.createBottomPanel("markdown-preview-panel", $panel);
+
                 $panel.on("panelResizeUpdate", function (e, newSize) {
                     $iframe.attr("height", newSize);
                 });
+
+
+                //$panel.setHeight(800);
+                $panel.css("height", "82%");
```

Comment:
Why 82% how did we end up with the number? Will it produce an issue if some extension changes the top layout(Eg. Tabs extension)?

Reply:
The above code line does not contribute to panel height fix. Hence removed from codebase.

Reply:
Checked and verified after testing